athena-msk/src/main/java/com/amazonaws/athena/connectors/msk/serde/MskCsvDeserializer.java

```diff
@@ -71,15 +71,13 @@ public TopicResultSet deserialize(String topic, byte[] data)
                     ));
                 }
                 catch (Exception e) {
-                    LOGGER.error("MskCsvDeserializer: Error in castValue : while converting raw value to typed value");
-                    e.printStackTrace();
+                    LOGGER.error("MskCsvDeserializer: Error in castValue : while converting raw value to typed value", e.getMessage(), e);
```

Comment:
I don't think this will actually work, you are ending up using the wrong version of .error() (you're using the variadac object version for string formatting).

All you need to do is this (notice how its just (String, Exception):
```
LOGGER.error("MskCsvDeserializer: Error in castValue : while converting raw value to typed value", e);
```

See this for reference:
https://logging.apache.org/log4j/2.x/log4j-api/apidocs/org/apache/logging/log4j/Logger.html#error-java.lang.String-java.lang.Throwable-
```
void [error](https://logging.apache.org/log4j/2.x/log4j-api/apidocs/src-html/org/apache/logging/log4j/Logger.html#line.893)([String](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html?is-external=true) message,
           [Throwable](https://docs.oracle.com/javase/8/docs/api/java/lang/Throwable.html?is-external=true) throwable)

Logs a message at the [ERROR](https://logging.apache.org/log4j/2.x/log4j-api/apidocs/org/apache/logging/log4j/Level.html#ERROR) level including the stack trace of the [Throwable](https://docs.oracle.com/javase/8/docs/api/java/lang/Throwable.html?is-external=true) throwable passed as parameter.

Parameters:
    message - the message object to log.
    throwable - the Throwable to log, including its stack trace.
```